src/openforms/formio/api/serializers.py

```diff
@@ -25,6 +25,13 @@ class TemporaryFileUploadSerializer(serializers.Serializer):
         use_url=False,
         validators=[MimeTypeValidator(), NoVirusValidator()],
     )
+    submission = serializers.SlugRelatedField(
+        slug_field="uuid",
+        queryset=Submission.objects.filter(completed_on=None),
+        label=_("Submission"),
+        write_only=True,
+        required=True,
+    )
```

Comment:
we use hyperlinked model serializers everywhere else, please use a hyperlink instead of just the UUID.